website/docs/validating-admission-policy.md

```diff
@@ -54,22 +54,9 @@ Find out more about different [enforcement points](enforcement-points.md)
     runtimeConfig:
       admissionregistration.k8s.io/v1beta1: true
     ```
-- Set `--experimental-enable-k8s-native-validation` in Gatekeeper deployments, or `enableK8sNativeValidation=true` if using Helm.
 
 ## Get started
 
-### Option 1: Install with Helm
-Update the `enableK8sNativeValidation` parameter in values.yaml or set during deployment
-- Enable the K8s Native Validating driver to allow users to create CEL-based rules in addition to the OPA driver and rego rules (alpha feature). Default is `false`
-```shell
-helm install gatekeeper/gatekeeper --name-template=gatekeeper --namespace gatekeeper-system --create-namespace \
-    --set enableK8sNativeValidation=true
-```
-
-### Option 2: Install with Gatekeeper deployment
-Edit the applicable deployments (`controller-manager` and `audit`), and update the following commandline flags:
-- Set `--experimental-enable-k8s-native-validation=true`
-
```

Comment:
Since we are defaulting to `true` I removed this doc section. However, we should add `experimental-enable-k8s-native-validation` is `beta` and is turned on by default somewhere in the docs.

@ritazh @maxsmythe @sozercan any suggestion on place on the doc for this^^?

Reply:
Added a note above in the same file, here - https://github.com/open-policy-agent/gatekeeper/pull/3476/files#diff-6f51397980ac58fff52f433210aa369e522c495bdd049d01ae538eaa3b9118b1R43

Reply:
you should mention the flag is beta, default to true, set to false to opt-out